src/labels/CustomLabels.labels

```diff
@@ -5365,6 +5477,32 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Top of all Gift Entry pages</shortDescription>
         <value>Gift Entry</value>
     </labels>
+    <labels>
+        <fullName>geHeaderMatchingReviewDonations</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for modal to review donations</shortDescription>
+        <value>Review Donations for {0}</value>
```

Comment:
@psadie-sfdo do you need additional explanation here for what {0} represents?

Reply:
Yes definitely! Added that in.